frontend/src/citizen-frontend/income-statements/IncomeStatementView.tsx

```diff
@@ -0,0 +1,362 @@
+            )}
+            {entrepreneur.lightEntrepreneur && (
+              <Item>{t.income.view.lightEntrepreneur}</Item>
+            )}
+          </>
+        }
+      />
+      <Row label={t.income.view.incomeSource} value="" />
+      {entrepreneur.selfEmployed && (
+        <Row
+          light
+          label={t.income.view.selfEmployed}
+          value={
+            <>
+              {entrepreneur.selfEmployed.attachments && (
+                <Item>{t.income.view.selfEmployedAttachments}</Item>
+              )}
+              {entrepreneur.selfEmployed.estimatedIncome && (
+                <EstimatedIncome
+                  estimatedIncome={entrepreneur.selfEmployed.estimatedIncome}
+                />
+              )}
+            </>
+          }
+        />
+      )}
+      {entrepreneur.limitedCompany && (
+        <Row
+          light
+          label={t.income.view.limitedCompany}
+          value={
+            entrepreneur.limitedCompany.incomeSource === 'INCOMES_REGISTER'
+              ? t.income.view.limitedCompanyIncomesRegister
+              : t.income.view.limitedCompanyAttachments
+          }
+        />
+      )}
+      {entrepreneur.partnership && (
+        <Row
+          light
+          label={t.income.view.partnership}
+          value={t.income.view.attachments}
+        />
+      )}
+      {entrepreneur.lightEntrepreneur && (
+        <Row
+          light
+          label={t.income.view.lightEntrepreneur}
+          value={t.income.view.attachments}
+        />
+      )}
+      {entrepreneur.accountant && (
+        <AccountantInfo accountant={entrepreneur.accountant} />
+      )}
+    </>
+  )
+}
+
+function AccountantInfo({ accountant }: { accountant: Accountant }) {
+  const t = useTranslation()
+  return (
+    <>
+      <H3>{t.income.view.accountantTitle}</H3>
+      <Row label={t.income.view.accountant} value={accountant.name} />
+      <Row label={t.income.view.email} value={accountant.email} />
+      <Row label={t.income.view.phone} value={accountant.phone} />
+      <Row label={t.income.view.address} value={accountant.address} />
+    </>
+  )
+}
+
+function CitizenAttachments({ attachments }: { attachments: Attachment[] }) {
+  const t = useTranslation()
+  return (
+    <>
+      <H2>{t.income.view.citizenAttachments.title}</H2>
+      {attachments.length === 0 ? (
+        <p>{t.income.view.citizenAttachments.noAttachments}</p>
+      ) : (
+        <Row
+          label={`${t.income.view.attachments}:`}
+          value={<UploadedFiles files={attachments} />}
+        />
+      )}
+    </>
+  )
+}
+
+function UploadedFiles({ files }: { files: Attachment[] }) {
+  return (
+    <FixedSpaceColumn>
+      {files.map((file) => (
+        <div key={file.id}>
+          <FileIcon icon={fileIcon(file)} />
+          <FileDownloadButton
+            file={file}
+            fileFetchFn={getAttachmentBlob}
+            onFileUnavailable={() => undefined}
```

Comment:
How unlikely is it that a user would navigate to this view within, say, 5 min of uploading a new attachment? Not that there aren't other cases where attachments might be unavailable.